docs/java/how-to-develop-a-worker-program-in-java.md

```diff
@@ -18,7 +18,9 @@ Then register all Workflow Types and all Activity Types that the Worker can exec
 For Activities, since they are stateless and thread-safe, instances are used in the registration process instead of the Java Class.
 A Worker can be registered with just Workflows, just Activities, or both.
 
-This single Worker Process can contain many Workers Objects.
+## Create Worker
+
+A single Worker Process can contain many Worker Objects.
```

Comment:
Let's use Worker Entity - and link to it
Link to Worker Process

Reply:
done